autopush/router/webpush.py

```diff
@@ -64,16 +68,18 @@ def preflight_check(self, uaid_data, channel_id):
         """Verifies this routing call can be done successfully"""
         uaid = uaid_data["uaid"]
         if 'current_month' not in uaid_data:
-            self.log.info("Record missing 'current_month' {record}",
-                          record=repr(uaid_data))
+            self.log.info(format="Dropping User", code=102,
```

Comment:
I numbered every code uniquely so we could identify where in the code-base we executed it from. The reason in a few cases is the same or similar, so knowing the origin of the call seemed more important than potentially the same reason in multiple places.